File: Heat/ViewModels/Customers/CustomerViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels
    Public Class CustomerViewModel
        Property ID As Integer
        <Display(name:="Nome")> _
        Property Name As String

        <Display(name:="Indirizzo")> _
        Property Address As String

        <Display(name:="Telefono")> _
        Property Telephone As String
    End Class
End Namespace

File: Heat.ConvertedToC#/ViewModels/Customers/CustomerViewModel.cs
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels
{
    public class CustomerViewModel
	{
		public int ID { get; set; }
		[Display(Name = "Nome")]
		public string Name { get; set; }

		[Display(Name = "Indirizzo")]
		public string Address { get; set; }

		[Display(Name = "Telefono")]
		public string Telephone { get; set; }
	}
}

